tests/unit/packaging/test_models.py

```diff
@@ -430,6 +430,14 @@ def test_acl(self, db_session):
             ("https://google.com/pypi/warehouse/tree/main", None),
             ("https://google.com", None),
             ("incorrect url", None),
+            (
+                "https://www.github.com/pypi/warehouse.git",
+                "https://api.github.com/repos/pypi/warehouse",
+            ),
+            (
+                "https://www.github.com/pypi/warehouse.git/",
```

Comment:
This duplicates the new parametrized example twice, no?

Reply:
Oh never mind I see, trailing slash, ignore me.

Reply:
Oof, I missed this too 😅

Reply:
I sorta wondered about adding additional incorrectly-formatted GitHub test URLs, but then I didn't want to overspecify or add test URLs that shouldn't occur as input.  The trailing slash was a subtle difference there though, yep :)